lua/bqf/preview/handler.lua

```diff
@@ -353,8 +353,9 @@ function M.redrawWin()
     local bufnr = tonumber(fn.expand('<abuf>')) or api.nvim_get_current_buf()
     local qwinid = fn.bufwinid(bufnr)
     if utils.isWinValid(qwinid) then
+        local preview = pvs.validate()
         M.close(qwinid)
-        M.open(qwinid)
+        if autoPreview or preview then M.open(qwinid) end
```

Comment:
I think that it should be `or` to ensure that if the preview will still show if it was toggled manually with ptoggleitem (while autoPreview) is disabled. Sorry for the formatting, I just fixed it.

Reply:
> I think that it should be `or` to ensure that if the preview will still show if it was toggled manually with ptoggleitem (while autoPreview) is disabled.

No, when user type `p`, even after doing resize window, should respect the last action.

Reply:
Oh, I see what you mean. Now that I think about it, there are two cases two consider when manually toggling (hitting `p`) – 1) when manually hiding the preview while autopreview is on, and 2) when manually showing the preview when autopreview is off.

Having `and` makes it so that the preview isn't reopened if for case 2) while or makes it so that the preview is incorrectly shown for case 1). I added an additional check to resolve both cases.

Reply:
`autoPreview` is changed when user type `p`, single `and ` logic is enough.

Reply:
Sorry, I'm not too familiar with the codebase. Upon testing, the preview isn't reopened upon resizing for case 2.
![Screen Recording 2023-09-11 at 3 21 42 AM](https://github.com/kevinhwang91/nvim-bqf/assets/56745535/0ca3acda-e5d9-46e7-b82f-e279b0d46f9a)

Just to confirm, does `pvs.validate()` give the state of the preview (i.e. whether it's open or not) at the time that is called?

Reply:
> autoPreview is changed when user type p, single and  logic is enough.

only change when type `P`.

just check the `pvs.validate()` is enough.

Reply:
Alright it should be fixed now